mongodb-atlas-mixin/dashboards/mongodb-atlas-sharding-overview.libsonnet

```diff
@@ -0,0 +1,1374 @@
+      datasource=promDatasource,
+      legendFormat='{{instance}} - find',
+      format='time_series',
+      interval='1m',
+    ),
+    prometheus.target(
+      'increase(mongodb_shardingStatistics_numHostsTargeted_insert_unsharded{job=~"$job",cl_name=~"$cl_name",rs_nm=~"$rs",instance=~"$instance"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - insert',
+      format='time_series',
+      interval='1m',
+    ),
+    prometheus.target(
+      'increase(mongodb_shardingStatistics_numHostsTargeted_update_unsharded{job=~"$job",cl_name=~"$cl_name",rs_nm=~"$rs",instance=~"$instance"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - update',
+      format='time_series',
+      interval='1m',
+    ),
+    prometheus.target(
+      'increase(mongodb_shardingStatistics_numHostsTargeted_delete_unsharded{job=~"$job",cl_name=~"$cl_name",rs_nm=~"$rs",instance=~"$instance"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - delete',
+      format='time_series',
+      interval='1m',
+    ),
+    prometheus.target(
+      'increase(mongodb_shardingStatistics_numHostsTargeted_aggregate_unsharded{job=~"$job",cl_name=~"$cl_name",rs_nm=~"$rs",instance=~"$instance"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - aggregate',
+      format='time_series',
+      interval='1m',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Unsharded',
+  description: 'The number of CRUD operations and aggregation commands run on an unsharded collection.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 10,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [
+        'min',
+        'max',
+        'mean',
+      ],
+      displayMode: 'table',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+
+{
+  grafanaDashboards+:: {
+    'mongodb-atlas-sharding-overview.json':
```

Comment:
Commit [56ff7ba](https://github.com/grafana/jsonnet-libs/pull/1044/commits/56ff7ba36996a51016f8e547b0038b0750b9198f) removed the if check for this dashboard, need to put it in manually again. Also, this is missing a dashboard link.

Reply:
oh yes thank you, good call, I had to reconvert the json to libsonnet bc dashboards_out was having an issue with the microseconds symbol